### ISSUE TITLE
crnn acc =0

### ISSUE THREAD

**Issue by ekils** [Holmeyoung/crnn-pytorch#41]
Hi, sorry for the bothering, but I'm facing the problem.
All my accuracy during validation are 0 , that is really weird.
Have you ever facing this problem during training?
I just used your pre-trained model and with only 3000 pics of  training set, because I only wanna quick check this model is work for me or not. If it works, I will put more dataset for my personal training.
My lr = 0.00005 and 0.0001 , both tried.
Image size = 120*32  with four  Chinese characters.
keep_ratio =False

**Comment by Holmeyoung** [comment]
Hi, the images used in pre-trained model are generated by me and they all look very well. Just try to make more training data, it will work!

**Comment by ekils** [comment]
Thanks for the reply.
It really work after with more data and with more patient :) 

I have another question :
While in training , all my input image size is (120,32) and in validation I saw there  has a code 
with "dataset.resizeNormalize(params.imgW,params.imgH)" . And in params, I set keep_ratio =True ,and imgW=120 , imgH=32. After several epochs training, my val acc comes to 0.96.

But when I put images which  from CTPN model (different size of images) and send them  to demo.py, the acc are terrible. 

So, it means I have to put my training data with different size? (e.g: (300.32), (150,32), (500,32).....    )
But  the code "dataset.resizeNormalize(params.imgW,params.imgH)"  will resize  and image will become distortion. What dataset  did  you make ?

**Comment by Holmeyoung** [comment]
Hi, the images from CTPN should look like your training data. If your training data is generated by yourself and looks very nice, while the images out from CTPN are more real, the net will do bad

**Comment by ekils** [comment]
So, you mean if I train with generated images and test with CTPN's crop image,  then the result will be very bad.... ?
I have to train with self crop real images and test with  TPN's crop images?
So  the 3600 thousand dataset is useless?

**Comment by wfdubowen** [comment]
Hello, I encountered the same problem, I used 6 million pieces of data for training. However, during verification, the predicted data is empty, and the previously generated data does not have this problem.
